tests/test_alerter.py

```diff
@@ -12,14 +13,23 @@
 from simplemonitor.Alerters import alerter, sns
 from simplemonitor.Monitors import monitor
 
+# Create a consistent "local" timezone and offset for the tests, for tests that
+# compare the offset between UTC and local time. For simplicity and
+# predictability, use a time zone that doesn't have daylight savings.
+LOCAL_TZ = "MST"
+LOCAL_TZ_OFFSET = -7
```

Comment:
I just defined these as top level constants vs. in the classes, mostly so that it would be easier to use the `@freeze_time` decorator in more places.